falcon/request.py

```diff
@@ -1394,16 +1424,26 @@ class RequestOptions(object):
             forward slash. However, this behavior can be problematic in
             certain cases, such as when working with authentication
             schemes that employ URL-based signatures.
+
+        default_media_type (str): The default media-type to use when
+            deserializing a response.
+
+        media_handlers (Handlers): A dict-like object that allows for you to
+            configure the media-types that you would like to handle.
     """
     __slots__ = (
         'keep_blank_qs_values',
         'auto_parse_form_urlencoded',
         'auto_parse_qs_csv',
         'strip_url_path_trailing_slash',
+        'default_media_type',
+        'media_handlers',
     )
 
     def __init__(self):
         self.keep_blank_qs_values = False
         self.auto_parse_form_urlencoded = False
         self.auto_parse_qs_csv = True
         self.strip_url_path_trailing_slash = True
+        self.default_media_type = 'application/json'
```

Comment:
@BenjamenMeyer Interestingly enough Falcon has set a default media type for headers for quite some time. It happens when a new instance of API is created. So this default value is never really used as it's usually set by the API class. However, I agree, this should probably be None or as @kgriffs suggested in one of his comments the `DEFAULT_MEDIA_TYPE` value.